### ISSUE TITLE
Completely black UI elements after latest beta client update

### ISSUE THREAD

**Issue by 414n** [ValveSoftware/steam-for-linux#8035]
#### Your system information

* Steam client version (build number or date): 1630453407 (Aug 31 2021, 20:58:17)
* Distribution (e.g. Ubuntu): Slackware64 14.2
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

After installing the latest beta client update today and restarting Steam, I was greeted with completely black UI elements, specifically the "main" elements in every section of the application.

These are screenshot of the issue, taken for the "Store" and "Library" sections, but the others are the same:

![image](https://user-images.githubusercontent.com/11655692/131752202-cf7d5a8f-cbc0-4799-bcee-de3af4243abd.png)
![image](https://user-images.githubusercontent.com/11655692/131752264-493f3aca-8cd8-4ba0-867f-ad7469d84b48.png)

Menu items and secondary windows (like the "Settings" window) work ok.
Big Picture mode also works and is now the only way I have to see something in Steam.

Before restarting steam, I saw in the [incoming changes](https://steamcommunity.com/groups/SteamClientBeta/announcements/detail/2956038687124632735) that there was a fix for issue #7935, which should fix a really similar issue for users with a system-wide freetype-2.11.x library. This is clearly not my case, as my system sports freetype-2.6.3, but maybe the fix applied for #7935 broke something for previous freetype versions.

When launching steam from the terminal, i can see a bunch of `src/vgui2/vgui_surfacelib/FontManager.cpp (287) : Assertion Failed: descs.Count() >= 1` messages. Don't know if this can be relevant or not.

#### Steps for reproducing this issue:

1. start the aforementioned client beta version

**Comment by kisak-valve** [comment]
Hello @414n, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by 414n** [comment]
Here it is: [gist](https://gist.github.com/414n/34891616266f2d4e4489173e8bbee632).
I added it to the opening message too.

**Comment by kisak-valve** [comment]
Thanks, I suspect that the missing / failed library pinning is the key detail here (https://gist.github.com/414n/34891616266f2d4e4489173e8bbee632#file-steam-info-txt-L97-L113)

**Comment by 4164696f73** [comment]
I've got the same issue. Can't find a way to solve it at all.
Here's my gist in case needed: https://gist.github.com/4164696f73/fed821c52aa948de1f92f2ed9d0da2c8

**Comment by mikajed** [comment]
Got the same issue but I did not opted in for the beta client.

I use Arch, everything is updated.

Here is the [Gist](https://gist.github.com/mikajed/69d61cd53f9e17399b05410f21190fcd) if needed

**Comment by salmonmolon** [comment]
Seems like a duplicate of #7935. Try downgrading freetype2 back to 2.10.4

**Comment by 4164696f73** [comment]
> Seems like a duplicate of #7935. Try downgrading freetype2 back to 2.10.4

Just tried it out (here's link to the command I've used: https://github.com/ValveSoftware/steam-for-linux/issues/7935#issuecomment-886026139) and it did fix the issues.

**Comment by same-jame** [comment]
> Got the same issue but I did not opt in for the beta client.
> 
> I use Arch, everything is updated.
> 
> Here is the [Gist](https://gist.github.com/mikajed/69d61cd53f9e17399b05410f21190fcd) if needed

Switching to the beta fixed the issue for me on Arch. (freetype2 2.11.0-3)

**Comment by kisak-valve** [comment]
Hello @4164696f73, @same-jame. since you're both on Arch, your issue is unrelated to this issue report. The issue with Freetype 2.11.0 on distros like Arch and Fedora 34 is being tracked at #7935 and should already be resolved in the [2021-08-31](https://steamcommunity.com/groups/SteamClientBeta/announcements/detail/2956038687124632735) Steam client beta update.

**Comment by smcv** [comment]
@414n, please could you run Steam from a terminal (Konsole, xterm or equivalent) and put the output in a gist?

**Comment by smcv** [comment]
> I suspect that the missing / failed library pinning is the key detail here

I agree, something is going badly wrong with the `setup.sh`/`run.sh` pair. The UI and games probably worked for you before by pure luck, but the changes made in the beta to fix the `steamwebhelper` on very recent distros (like Arch and Fedora) make it more important that the library pinning is done correctly.
